itest/gen_protos.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright (C) 2015-2022 Lightning Labs and The Lightning Network Developers
+# Copyright (C) 2022 Bottlepay and The Lightning Network Developers
```

Comment:
2023 😭

Reply:
Fixed in https://github.com/bottlepay/lndsigner/compare/c8d4e2145dcf4ce6412f5b53ebd0f5b7422f79f4..2471acc1aef216941024a860638e0b80789ab598 :sob: